src/BenchmarkDotNet/BenchmarkDotNet.csproj

```diff
@@ -13,14 +13,13 @@
   </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="Templates\*" Exclude="bin\**;obj\**;**\*.xproj;packages\**;@(EmbeddedResource)" />
-    <EmbeddedResource Include="Environments\microarchitectures.txt" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
     <PackageReference Include="Gee.External.Capstone" Version="2.3.0" />
     <PackageReference Include="Iced" Version="1.17.0" />
     <PackageReference Include="Microsoft.Diagnostics.Runtime" Version="2.2.332302" />
-    <PackageReference Include="Perfolizer" Version="[0.3.17]" />
+    <PackageReference Include="Perfolizer" Version="[0.4.0]" />
```

Comment:
Just for my education: why is the version `[0.4.0]` rather than just `0.4.0`?

Reply:
#2358